prow/cmd/deck/static/user-script.js

```diff
@@ -172,15 +172,30 @@ function createPRCardTitle(pr, tidePools, jobStatus, mergeAbility) {
 function createList(list, itemStyle = []) {
     const container = document.createElement("UL");
     container.classList.add("mdl-list", "job-list");
-
+    const getStateIcon = (state) => {
+        switch (state) {
+            case "success":
+                return "check_circle";
+            case "failure":
+                return "error";
+            case "pending":
+                return "watch_later";
+            case "triggered":
+                return "access_time";
```

Comment:
I think this is already out of sync with the other script

Reply:
done